rbac/management/group/view.py

```diff
@@ -255,6 +259,8 @@ def update(self, request, *args, **kwargs):
                 "name": "GroupA"
             }
         """
+        group = get_object_or_404(Group, uuid=kwargs.get('uuid'))
```

Comment:
Nit: could we put this into the function just use uuid as input? And do you mind adding a unit test for the update, just like the one you did for delete?

Reply:
Sure! That seems reasonable. I realized after opening this PR that I'd forgotten a test. I added one in a later commit but it seems that didn't get properly added into the PR. I'll get it re-added. Thanks!